portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile/MyTestFile.testcase

```diff
@@ -0,0 +1,31 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+property testray.main.component.name = "Forms";
+setUp {
+TestCase.setUpPortalInstance();
+User.firstLoginPG();
+Navigator.openURL();
+}
+tearDown {
+User.logoutAndLoginPG(
+userLoginEmailAddress = "test@liferay.com",
+userLoginFullName = "Test Test");
+}
+
+@description = "Access Portal Global Menu and validate CONFIGURATION string at
+Control Panel"
+@priority = "5"
+test ValidateConfigurationStringAtControlPanel {
```

Comment:
According with the [ticket](https://liferay.atlassian.net/browse/LRQA-82594), the name should be **AssertConfigurationSectionPresentInControlPanel**